php/class-wp-cli.php

```diff
@@ -135,7 +137,7 @@ public function create_terms_for_posts(): void {
 	 * @subcommand create-author-terms-for-posts
 	 * @synopsis [--post-types=<csv>] [--post-statuses=<csv>] [--unbatched] [--records-per-batch=<records-per-batch>] [--specific-post-ids=<csv>] [--above-post-id=<above-post-id>] [--below-post-id=<below-post-id>]
 	 * @return void
-	 * @throws Exception If above-post-id is greater than or equal to below-post-id.
+	 * @throws Exception If above-post-id is greater than or equal to below-post-id, or if unable to obtain a prolific author account.
```

Comment:
Does it still throw an exception? If it does we should maybe adjust the "prolific author" part.

Reply:
Good catch